.github/workflows/stale.yml

```diff
@@ -0,0 +1,29 @@
+name: Mark stale issues and pull requests
+
+on:
+  schedule:
+  - cron: "30 1 * * *"
+  workflow_dispatch:
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+    steps:
+    - uses: actions/stale@v3
+      with:
+        operations-per-run: 1000
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        stale-issue-message: 'This issue is stale because it has been open 18 months with no activity. Remove stale label or comment or this will be closed in 5 days.'
```

Comment:
Overall I'm in favor of cleaning out very stale issues.  I wonder if we might want to work on this message a bit. 

For a few projects that started using this bot that I know if, it was kinda a bummer to people who had spent time writing up an issue to get a terse message on a dozen or so issues when it didn't really seem like _their_ fault that the issue had gone stale.

Reply:
+1.

Steven, would you like to provide a suggestion? Have you seen a project that's done this particularly well?

Reply:
Rust had a pretty lengthy message before they removed the stale bot (https://github.com/rust-lang/cargo/pull/6463/files#diff-76cc7f7adb754443d391eb9c8df1d4a47e8cd668179e41915153685a0b83b30aL14) 

That seems a bit long to me.  Perhaps something that is still short but a bit more conversational:

> We've marked this issue as stale because it has been inactive for 18 months. If this issue is still relevant, removing the stale label or adding a comment will keep it active.  Otherwise, we'll close it in 5 days to keep the issue queue tidy.  Thank you for your contribution to CockroachDB!

Reply:
I like it, updated.